### DISCUSSION TITLE
openDTU Fusion mit Home Assistant - Daten nach Abschalten der Wechselrichter

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2251] [Q&A]
Hallo,

ich habe eine openDTU Fusion mit HM-800 und HMS-1000 aktiv, läuft alles prima.

Nun habe ich zusätzlich Home Assistant (HA) und MQTT installiert und greife darüber auf die Daten zu, das läuft auch gut.

Die openDTU zeigt auch alle Daten dann noch weiterhin an, wenn die Wechselrichter sich abends abgeschaltet haben. Diese Daten sind wohl in der openDTU gespeichert.

HA allerdings hat dann keine Daten mehr, das scheint auf die direkten Daten der WR zuzugreifen, die in der openDTU ja nicht mehr empfangen werden können, wenn die WR abgeschaltet haben. Man kann im HA nun keine Daten mehr sehen (wie in der openDTU), das ist nicht so schön.

Gibt es eine Möglichkeit, die openDTU-Anzeigen in HA zu übernehmen?

Ich bin da eher Laie, kenne mich nicht wirklich aus.

Danke im Voraus und Grüße
Werner

**Comment by Tpaydt** [comment]
Hast du in der OpenDTU unter MQTT das Feld 
Retain Flag aktivieren auf aktiv gesetzt?

**Comment by Fokker-1** [comment]
Hallo,

danke für die Info.

Ja, ist gesetzt.

Gruß
Werner

**Reply by Tpaydt** [reply]
Evtl noch die Ablauffunktion deaktivieren unter mqtt

**Reply by Fokker-1** [reply]
Ja, danke, das war es!

Gruß
Werner
